src/openrct2-ui/windows/Windows.h

```diff
@@ -37,109 +41,175 @@ namespace OpenRCT2::Ui::Windows
     extern bool gWindowSceneryEyedropperEnabled;
     extern bool gDisableErrorWindowSound;
 
+    // About
```

Comment:
These comments are intended to quickly search or scan for signatures for a particular file.